check_tags_in_requests.py

```diff
@@ -77,32 +77,33 @@ def isNewPackage(self, tgt_project, tgt_package):
         return False
 
     def checkTagInRequest(self, req, a):
-        is_new = False
         u = osc.core.makeurl(self.apiurl,
-                             ['source', a.tgt_project, a.tgt_package],
+                             ['source', a.src_project, a.src_package],
                              {'cmd': 'diff',
                               'onlyissues': '1',
                               'view': 'xml',
-                              'opackage': a.src_package,
-                              'oproject': a.src_project,
-                              'orev': a.src_rev})
+                              'opackage': a.tgt_package,
+                              'oproject': a.tgt_project,
+                              'rev': a.src_rev})
         try:
             f = osc.core.http_POST(u)
         except (HTTPError, URLError):
-            is_new = self.isNewPackage(a.tgt_project, a.tgt_package)
-        
-        # in case the quest have not the matched revision in Factory
-        # and it is a new package to target project, then leave it to
-        # human review
-        if is_new:
-            self.logger.info("New package to %s and have not the matched revision in Factory"%a.tgt_project)
+            if self.isNewPackage(a.tgt_project, a.tgt_package):
+                self.review_messages['accepted'] = 'New package'
+                return True
+
+            self.review_messages['accepted'] = 'Warning: failed to load package diff'
             return True
```

Comment:
yes, if we return None the review may stay open forever until someone overrides it or fixes the cause. I fear that if we return True right away the bot will approve requests on transient failures. Sometimes OBS is overloaded to create diffs and errors out IIRC. At a later time it may work again.